src/test/resources/ProfilingFilterTest/exp/topic1.dita

```diff
@@ -11,7 +11,8 @@
     <p class="- topic/p " platform="osx">OS X platform</p>
     <p class="- topic/p " platform="windows osx">Windows and OS X platform</p>
     <foreign class="- topic/foreign ">
-      <p platform="windows" class="- topic/p ">Foreign Windows</p>
+      <para>non-DITA para</para>
```

Comment:
Updated, adding one without class, and one with non-DITA class, both with `platform="windows"`